tests/unit_tests/test_ipmi.py

```diff
@@ -11,7 +11,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from mock import patch, MagicMock
+try:
```

Comment:
if unittest.mock provides the same functionality, then let us use that and remove the `from mock import patch, MagicMock` all together. No need to do try and except.

Reply:
it's already removed vikrant, i don changes only for "test_imcversion.py"